multicluster/config/samples/clusterset_init/multicluster_membercluster_template.yaml

```diff
@@ -25,5 +25,6 @@ spec:
       secret: leader-access-token
       server: https://<LEADER_CLUSTER_IP>:6443
   members:
-    - clusterID: test-cluster-east
-  namespace: antrea-mcs-ns
+    - clusterID: test-cluster-leader
+    - clusterID: test-cluster-member
```

Comment:
@hjiajing  Could you help to review these changes in templates? I hope it has minor impact to CI, thanks.

Reply:
@luolanzone : please update the ObjectMeta.Name first.

Reply:
sure, I  am working the diagram, will submit a new commit once is done.